tools/ClassGen/InstrBuilder.cpp

```diff
@@ -239,6 +239,13 @@ void InstrBuilder::emitClass(std::ostream &os) const {
         }
         break;
       }
+      case VerifyKind::TypeCheck: {
```

Comment:
We could use a comment here that explains what code is being generated. Same for the other cases in this switch.